vividus-extension-web-app/src/test/java/org/vividus/ui/web/configuration/DomainManagerTests.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019-2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.ui.web.configuration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.vividus.ui.web.action.DomainManager;
+
+public class DomainManagerTests
+{
+    @ParameterizedTest
+    @CsvSource({
+            "https://www.domain.com, .domain.com",
+            "http://127.0.0.1:8080, 127.0.0.1",
+            "http://localhost:8080,  localhost"
+    })
+    void shouldAddCookie(String urlAsString, String domain)
```

Comment:
please update method name to reflect the actual test

Reply:
updated, thanks